ergo-rest/src/wasm_timer/timer/ext.rs

```diff
@@ -23,30 +23,6 @@ pub trait TryFutureExt: TryFuture + Sized {
     /// If the future completes before `dur` elapses then the future will
     /// resolve with that item. Otherwise the future will resolve to an error
     /// once `dur` has elapsed.
-    ///
```

Comment:
I deleted this doctest since otherwise we'd leak some `wasm-timer` internals through `ergo-rest`.